README.rst

```diff
@@ -1,3 +1,25 @@
+.. Copyright (C) 2023 ANSYS
+.. SPDX-License-Identifier: MIT
+..
+..
+.. Permission is hereby granted, free of charge, to any person obtaining a copy
+.. of this software and associated documentation files (the "Software"), to deal
+.. in the Software without restriction, including without limitation the rights
+.. to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+.. copies of the Software, and to permit persons to whom the Software is
+.. furnished to do so, subject to the following conditions:
+..
+.. The above copyright notice and this permission notice shall be included in all
+.. copies or substantial portions of the Software.
+..
+.. THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.. IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.. FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+.. AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.. LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+.. OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+.. SOFTWARE.
+
```

Comment:
For most of our products, this is located at the bottom of the READMe file.
Should it be the same here? @jorgepiloto @RobPasMue 
It could mean that we have to double check with the legal department.

Reply:
You are rigth.
I got confused with the section "THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARR..." that we usually add at the end.
But Indeed it is just a comments. It was obvious.

Reply:
Legal asked to have this at the very top of the file, following the SPDX guidelines.